src/opm/parser/eclipse/EclipseState/Schedule/TimeMap.cpp

```diff
@@ -246,8 +246,13 @@ namespace {
             }
         }
 
+        // Accept lower- and mixed-case month names.
+        std::string monthname = monthItem.get<std::string>(0);
```

Comment:
```
#include <opm/parser/eclipse/Utility/String.hpp>
...
std::string monthname = Uppercase(monthItem.get<std::string>(0));
```

Reply:
you owe me a coke with cap.

Reply:
Hehe - I don't do coke but I'd be happy to buy you a :beer: